tutorials/java/chapters/ch07.adoc

```diff
@@ -0,0 +1,221 @@
+:imagesdir: images
```

Comment:
This chapter feels like a wildfly tutorial :) might be a little unnecessary for non-wildfly users?